src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -111,7 +111,7 @@
 
     <!-- Append the xunit trait options to the end of the TestCommandLine -->
```

Comment:
Can you change this comment to something like "Insert trait options into the TestCommandLine"?